src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty/config.jelly

```diff
@@ -14,7 +14,8 @@
 					<j:otherwise>
 						<j:forEach var="permission" items="${effectivePermissions}">
 							<f:entry field="${permission.formName}">
-								<kube:checkbox title="${permission.name}" checked="${permission.granted}" readonly="${permission.readonly}"/>
+								<j:set var="readOnlyMode" value="${permission.readonly}"/>
+								<f:checkbox title="${permission.name}" checked="${permission.granted}" />
```

Comment:
```suggestion
								<f:checkbox title="${permission.name}" checked="${permission.granted}"  readonly="${permission.readonly}"/>
```

does not work?

Reply:
That only prevents clicking through JavaScript it doesn’t set the standard DOM property for it and it doesn’t change the UI to show they can’t click it.

At least from reading the code I haven’t tried it...

Reply:
Looks like I was confusing `readonly`, old, with `readOnlyMode` introduced in https://github.com/jenkinsci/jenkins/pull/4506. It seems the latter is not an attribute but an (undocumented?) dynamically-scoped variable.

Reply:
It’s documented on Jenkins.io https://www.jenkins.io/doc/developer/views/read-only/#enabling-read-only-view-support

Reply:
I see. Can you link to that from the attribute docs for `readonly` on affected controls please?

Reply:
it's not an attribute though, would you expect it as part of the top level description?

Reply:
I am just guessing someone trying to figure out how to make a checkbox read-only is going to look at the documentation for the attribute named `readonly`, so put it there.